### DISCUSSION TITLE
Migrate to New SLZB-06 coordinator

### DISCUSSION THREAD

**Discussion by mjhcdk** [Koenkk/zigbee2mqtt#24992] [Devices]
Hi, 

I have the CC2652RB USB Coordinator from slae.sh , but want to migrate to network based device solution using   ZMLIGHT SLZB-06p7. As both devices use texas  CC2652 chipset, I believe i have read somewhere its possible to change adaptor without re-paring all end-points, but obviously unable to find info now.  Anyone who have done this succesfully, and can share some guidance.  Thanks

**Comment by terierul** [comment]
If you are switching from another CC26xx Zigbee adapter, in most cases, you do not need to repair your Zigbee devices if you follow these steps:

1. Shut down Zigbee2MQTT/ZHA and power off your previous Zigbee coordinator.
2. Power off all your Zigbee routers for at least 15 seconds. You can do this at once by turning off the electricity in the whole building.
3. Power on back all your Zigbee routers.
4. Power on your new SLZB-06/06p7 or 06p10 adapter.
5. Start your Zigbee2MQTT or ZHA.
6. It may take some time (e.g., 5-10 minutes) for your Zigbee devices to come back online again with the new Zigbee coordinator.
If this does not work, please change the IEEE address of this Coordinator to match your previous Coordinator and repeat steps 1 to 5.
